app/templates/eventView.html

```diff
@@ -29,82 +29,94 @@ <h1 id="title">{{page_title}}</h1>
+                    {% else %}
+                        <p class="heading-size"><b>{{eventData.name}}</b> will take place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and will be located in <b>{{eventData.location}}</b>.</p>
+                    {% endif %}
+                    <p class="size">
+                        {{eventData.description}}<br><br>
+                    </p>
+
+                    <div class="size">
                         {% if eventData.isService %}
                             This event earns service hours.
+                            <br />
                         {% else %}
                             This event does not earn service hours.
+                            <br />
                         {% endif %}
                         {% if eventData.isRsvpRequired %}
                             This event requires RSVP.
+                            <br />
                         {% else %}
                             This event does not require RSVP.
+                            <br />
                         {% endif %}
                         {% if eventData.recurringId %}
                             This event is a recurring event.
+                             <br />
                         {% endif %}
                         {% if eventData.isTraining %}
                             This event is a training.
+                            <br />
                         {% endif %}
                     </div>
                 </div>
-            </div>
-            <div class="container">
-              <div class="row">
-                <div class="col-sm">
-                    <h3><br>Facilitators:</h3>
-                        {% if eventFacilitatorNames %}
-                            <ul>
-                                {% for facilitator in eventFacilitatorNames%}
-                                    <li>{{facilitator.firstName}} {{facilitator.lastName}}</li>
-                                {% endfor %}
-                            </ul>
-                        {% else %}
-                            <div>No facilitators.</div>
-                        {% endif %}
-                    </div>
-                    <div class="col-sm">
-                        {% if programManager %}
-                            <div><h3><br>Program Manager:</h3></div>
-                                <div>
-                                    {{programManager.user.firstName}} {{programManager.user.lastName}}<br>
-                                    {{programManager.user.email}}
-                                </div>
-                        {% endif %}
-                    </div>
-                </div>
-                <div>
-                    <div class="row">
+
+                <div class=" col-md-6">
```

Comment:
If you made it this far, might as well remove the added spaces in front of your col-md-6 's. I'm assuming you removed a class in front of it...